backend/tests/vscode-events.spec.ts

```diff
@@ -106,8 +106,11 @@ describe('vscode-events unit test', () => {
             }
             return events.performAction(item, 1);
         });
-        it("File as ActionType", () => { 
-            uriMock.expects("parse").withExactArgs("https://chakra-ui.com/alert").resolves();
+        it.skip("File as ActionType", () => { 
+            uriMock.expects("parse").withExactArgs("README").resolves();
+
```

Comment:
This doesn't test anything in our code --> uriMock.expects("parse") 
why is it in skip?

Reply:
this is Outdated, you need to refresh the pr